src/amber/server/env.cr

```diff
@@ -1,33 +1,38 @@
 module Amber
   def self.env
-    Env.new(Settings.env.not_nil!.downcase)
+    @@env ||= Env.new(ENV["AMBER_ENV"]? || "development") # Would be best to only define this once.
   end
 
   class Env
-    def initialize(@env : String | Symbol)
+    alias EnvType = String | Symbol 
+
+    def initialize(@env : String) end
+
+    def is?(environment : EnvType)
+      @env == environment.to_s.downcase
     end
 
-    def in?(environment_list : Array(String | Symbol))
-      (environment_list.map &.to_s.downcase).includes? @env
+    def in?(environment_list : Array(EnvType))
+      environment_list.each do |other_env|
+        return true if is?(other_env)
+      end
     end
 
-    def is?(environment : String | Symbol)
-      @env == environment.to_s.downcase
+    def in?(*environment_list : Object)
+      in?(environment_list.to_a)
```

Comment:
This is a nice addition

Reply:
You can remove `: Object`

Reply:
If you remove `: Object` it will respond to nil i.e, `env.in?()` rather than throwing a method not found. This will also cause it to error when I convert the tuple to an array.

Reply:
So by passing `: Object` it kinda makes the argument `required`, interesting. 

I would think the user of the method is responsible to not pass nil. Do you have an example of your comment above I can be learning something new today 😄